api/models/task.go

```diff
@@ -23,7 +23,7 @@ const (
 
 const (
 	// FormatDefault ...
-	FormatDefault = ""
+	FormatDefault = "default"
```

Comment:
If DefaultFormat is empty this condition will not work correctly:
https://github.com/iron-io/functions/pull/404/files#diff-1e508e75f470efdd06b4bf58edef75a2R348